src/composables/useNetwork.ts

```diff
@@ -27,6 +27,13 @@ const NETWORK_ID =
 if (windowAvailable) localStorage.setItem('networkId', NETWORK_ID.toString());
 export const networkSlug = config[NETWORK_ID].slug;
 export const networkConfig = config[NETWORK_ID];
+export const networksMap = {
```

Comment:
Since lots of things could be a 'networksMap' let's rename it 'networkLabelMap' to be more specific.